packages/components/src/summary/number.js

```diff
@@ -15,9 +15,10 @@ import { Text } from '@woocommerce/experimental';
 import Link from '../link';
 
 /**
- * A component to show a value, label, and an optional change percentage. Can also act as a link to a specific report focus.
+ * A component to show a value, label, optionals change percentage and children node. Can also act as a link to a specific report focus.
  *
  * @param {Object} props
+ * @param {number} props.children
```

Comment:
I suppose this would be `*`? As it could be a component, string, number, etc

Reply:
`JSX.Element` would be fine here I think. I don't think it should be a string or number?

Reply:
Yes, thank you. My google skillz weren't returning this

Reply:
Oh my bad, I forgot to change it! But I think it should be `{Node}`. What do you think?

Reply:
Oh my bad, I forgot to change it! Yes, I think `{Node}` would be right, is you are using in other components.

Reply:
Ha thats funny. Node is not correct so we're doing it wrong. I reckon go for React.ReactNode. We can fix up our other JS doc later 😄

Reply:
Also apologies that the existing code is a bit misleading here.

Reply:
No problem. I'll change it 🙂

Reply:
No problem. I'll change it 🙂

There is an issue, if I try to use `React.ReactNode` eslint throw `The type 'React' is undefined`, and if I import React then throw `'react' should be listed in the project's dependencies` 😅

How do you want to proceed?

Reply:
Good question. Elsewhere in the repo uses `Node`

https://github.com/woocommerce/woocommerce-admin/blob/d8a58509dd33c43dcfd97c8c1fb30b064997ac7a/packages/components/src/section/index.js#L38

>Node is not correct so we're doing it wrong

I suppose [Node refers to a DOM object](https://developer.mozilla.org/en-US/docs/Web/API/Node) and `React.ReactNode` is more appropriate. Since this is blocking the pull request, I think going with `Node` is best and we can address the issue later. FWIW, despite not being correct, in the context of WC Admin I fully get the intention and fixing this is low priority for me.

Reply:
Sorry I didn't reply sooner on this. Yes I agree with @psealock it was not my intention to make you spin your wheels on the node type here. Sorry about that! We can fix it later.